### PR TITLE
proof-read the docs / fix broken links and bump versions

### LINKED ISSUE #561: CSI docs needs to be updated for 1.27

### PR DESCRIPTION
**Pull request by coulof**
I proofread the entire doc website and propose to bump some versions & fix broken links.

The biggest change is on the Developing page to point on github over the google group.

resolves #561, #527

### CODE REVIEW

File: book/src/developing.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Remain Informed
 
-All developers of CSI drivers should join https://groups.google.com/forum/#!forum/container-storage-interface-drivers-announce to remain informed about changes to CSI or Kubernetes that may affect existing CSI drivers.
+All developers of CSI drivers should watch the repository [https://github.com/container-storage-interface/spec](https://github.com/container-storage-interface/spec) to remain informed about changes to CSI or Kubernetes that may affect existing CSI drivers.
```
[the remark this answers is not in the document]
Comment (coulof) [reply book/src/developing.md:6]: Should I point to https://github.com/container-storage-interface/spec/releases or revert back to the google group ?

[the remark this answers is not in the document]
Comment (xing-yang) [reply book/src/developing.md:6]: Please revert this back to the google groups.

[the remark this answers is not in the document]
Comment (coulof) [reply book/src/developing.md]: now reverted to original

**Pull request merged**